src/egon/data/datasets/fix_ehv_subnetworks.py

```diff
@@ -230,57 +225,87 @@ def fix_subnetworks(scn_name):
             6,
         )
 
-        # Missing 380kV line near Elsfleth
+        # Missing line from Umspannwerk Plottling to Gänsdorf UW
         add_line(
-            # Line
-            8.419326700000001,
-            53.229867000000006,
-            # Schaltanlage Elsfleth/West
-            8.402976949446648,
-            53.2371468322213,
+            12,
+            853318030996972,  # Umspannwerk Plottling
+            48,
+            76508569478315,
+            12,
+            769768646403532,  # Gänsdorf UW
+            48,
+            80533685376445,
```

Comment:
Looks a bit strange to me. Is the format really correct?

Reply:
There was an inconsistency with "." and ",". Corrected in a4dd369747b662e5d2eb5b4ff3bc84f6097a0f4d